src/main.ts

```diff
@@ -5,44 +5,91 @@ import '../prime.css';
  * and: https://bugs.webkit.org/show_bug.cgi?id=197960
  */
 import 'element-internals-polyfill';
-import { loadFonts } from './lib';
+import { SvelteComponent } from 'svelte';
+import { addStyles, loadFonts } from './lib';
 
 export { notify } from './lib/notify';
 
+/*
+ * `element` is only present if compiled with the `customElement` option, which it is. Type it such that it exists.
+ * https://github.com/sveltejs/svelte/blob/master/packages/svelte/src/runtime/internal/public.d.ts#L72-L73
+ */
+type ComponentWithElement = SvelteComponent & {
+  element: typeof HTMLElement;
+};
+
+const main = async () => {
+  // These components are exported as pure Svelte components.
+  const componentModules = {
+    'v-badge': await import('./elements/badge.svelte'),
```

Comment:
Can these become static imports since we removed the tag side effect?

Reply:
Yup, that's totally reasonable. We'll do that when we restructure the package.